kube_hunter/conf/parser.py

```diff
@@ -56,6 +56,13 @@ def parser_add_arguments(parser):
         help="Set log level, options are: debug, info, warn, none",
     )
 
+    parser.add_argument(
+        "--log-file",
+        type=str,
+        default=None,
+        help="Set logfile",
```

Comment:
It would be better to write something more indicative. like: "Path to a log file to output all logs to"

Reply:
Done.